src/molecules/Tabs.tsx

```diff
@@ -77,29 +137,36 @@ const Tabs = withRouter(({ match }) => {
               I'm description Lorem ipsum dolor sit amet consectetur adipisicing
               elit. Quo sapiente tenetur cum, quis, quia veniam quod fugit
               eligendi cupiditate vero, sequi asperiores cumque eos ipsum
-              pariatur alias dolore saepe unde.
+              pariatur alias dolore saepe unde. I'm description Lorem ipsum
+              dolor sit amet consectetur adipisicing elit. Quo sapiente tenetur
+              cum, quis, quia veniam quod fugit eligendi cupiditate vero, sequi
+              asperiores cumque eos ipsum pariatur alias dolore saepe unde.
             </StyledP>
           )}
         />
         <Route
           path={`${match.url}/info`}
           render={() => (
             <StyledP>
-              I'm info Lorem ipsum, dolor sit amet consectetur adipisicing elit.
-              Inventore dolorum similique officia ipsa magni, expedita
-              consequuntur pariatur perferendis ipsam at reprehenderit,
-              voluptate id, debitis amet facilis voluptatibus qui illo commodi.{' '}
+              <StyledTable>
+                {dataInfo.map(item => (
+                  <Table title={item.title} data={item.data} />
+                ))}
+              </StyledTable>
             </StyledP>
           )}
         />
         <Route
           path={`${match.url}/review`}
           render={() => (
             <StyledP>
-              I'm review Lorem ipsum dolor, sit amet consectetur adipisicing
-              elit. Quia inventore accusamus in suscipit minus praesentium
-              voluptate laboriosam exercitationem fugiat, perferendis illum. Et
-              ad dolore beatae consequuntur soluta animi amet ullam.
+              {dataReviews.map(review => (
```

Comment:
I'd move this into a separate component (`pages/product/ProductReviews.tsx`), where it would accept reviews and render a `Review` component for each review item:

```tsx
// pages/product/ProductReviews.tsx
import Review from './ProductReview'

const ProductReviews = ({ reviews }) => {
  return reviews.map((review) => <Review {...} />)
}
```

> The paths to files may be different, but those components **relate to the Product page**, so they must not be present in `molecules`. I think the same can be said about these `Tabs`.

So the components structure is the following:

```
- pages/
  - product/
    - index.tsx // the "/product" page, parent component
    - ProductReviews.tsx // the list of reviews
    - ProductReview.tsx // a SINGLE review component
```